src/main/java/org/apache/xml/security/keys/content/DEREncodedKeyValue.java

```diff
@@ -37,7 +37,9 @@
 public class DEREncodedKeyValue extends Signature11ElementProxy implements KeyInfoContent {
 
     /** JCA algorithm key types supported by this implementation. */
-    private static final String[] supportedKeyTypes = { "RSA", "DSA", "EC"};
+    private static final String[] supportedKeyTypes = { "RSA", "DSA", "EC",
+            "XDH", "X25519", "X448",
+            "EdDSA", "Ed25519,", "Ed448"};
```

Comment:
Is the trailing comma in `ED25519,` by purpose?

Reply:
Good catch, thanks Philip.